test/app/lib/cinematic/DialogSystem.spec.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-env jasmine */
+import { processText, wrapText } from '../../../../app/views/play/cinematic/DialogSystem'
+
+const dialogNode = text => ({ text })
+
+describe('dialogSystem', () => {
+  describe('text processing', () => {
```

Comment:
it looks like this is only for testing wrapText, can modify description here to specify that